src/schema.rs

```diff
@@ -435,3 +436,20 @@ pub struct CTExpectation {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub size: Option<u32>,
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub struct SynProxy {
+    pub family: NfFamily,
+    pub table: String,
+    pub name: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub handle: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    /// maximum segment size (must match your backend server)
+    pub mss: Option<u16>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    /// window scale (must match your backend server)
+    pub wscale: Option<u8>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub flags: Option<HashSet<SynProxyFlag>>,
```

Comment:
Could also be documented.

Reply:
Thanks, fixed in 855b72b8dfb9a3010a84c71c290a724e9a75743c